### ISSUE TITLE
OS info

### ISSUE THREAD

**Issue by N4S4** [ddvk/remarkable2-recovery#21]
Hi,

thanks for the beautiful instructions, even because customer support is the worse.
unfortunately i stopped the UI service by mistake so my remarkable is stuck on starting screen.
I know in the instructions says to use linux but by any chance do you know if linux wsl in windows works?

thank you

**Comment by Eeems** [comment]
Did you stop, or disable the service? If you stopped it, just hold the power button for 10+ seconds, and then power it on again. If you disabled the service, you should still be able to SSH in if you have the password or setup a SSH key, at which point you can just enable and start the service again.

**Comment by N4S4** [comment]
well i stopped and started another service which i installed via ssh but than i realized later that was not compatible....
 i do not have ssh pass anymore unfortunately

**Comment by Eeems** [comment]
> I know in the instructions says to use linux but by any chance do you know if linux wsl in windows works?

In theory, you might be able to get it to work, but you'd have to jump through hoops to get the USB device passed through, and you won't have the immediate feedback with dmesg that you'd get by running linux natively. I'd recommend just putting ubuntu onto a usb drive and live booting that: https://ubuntu.com/tutorials/install-ubuntu-desktop#3-create-a-bootable-usb-stick

**Comment by N4S4** [comment]
yeah i was thinking to boot a live distro and try...well I will try that definitely and revert back... thank you

**Comment by Eeems** [comment]
What instructions were you following that lead to this?

**Comment by N4S4** [comment]
it was during the installation of one of the launcher https://github.com/reHackable/awesome-reMarkable#take-special-care-if-you-are-using-a-remarkable-2  not sure which one was as i totally erased my memory for the nervous lol

**Comment by Eeems** [comment]
Ah, so it looks like you missed this portion right above it: https://github.com/reHackable/awesome-reMarkable#write-down-your-ssh-password

**Comment by N4S4** [comment]
indeed

**Comment by Eeems** [comment]
I've opened https://github.com/reHackable/awesome-reMarkable/pull/172 to make the warning section stand out more.

**Comment by N4S4** [comment]
Yep that is a great idea!
